miro_preprocessor/xml_to_json_run_task/src/xml_to_json_run_task.py

```diff
@@ -0,0 +1,61 @@
+# -*- encoding: utf-8 -*-
+
+"""
+Lambda to run the Miro XML to JSON task when receiving an S3 event.
+"""
+
+import os
+
+import boto3
+
+import s3_utils
+import sns_utils
+
+
+def command_for_xml_to_json_task(event, cluster_name, container_name, task_definition):
+    bucket_name = event["bucket_name"]
+    object_src = event["object_key"]
+    object_dst = f'{object_src}.txt'
+
+    cmd = [
+        f'--bucket={bucket_name}',
+        f'--src={object_src}',
+        f'--src={object_dst}'
+    ]
+
+    return {
+        "cluster_name": cluster_name,
+        "container_name": container_name,
+        "task_definition": task_definition,
+        "started_by": "xml_to_json_run_task",
+        "command": cmd,
+    }
+
+
+def post_to_sns(sns_client, topic_arn, cmd):
+    return sns_utils.publish_sns_message(
+        sns_client,
+        topic_arn,
+        cmd
+    )
+
+
+def main(event, _):
+    print(f'event = {event!r}')
+
+    topic_arn = os.environ["TOPIC_ARN"]
```

Comment:
How do these environment variables end up in the Lambda? I don’t see any associated Terraform config, or have I missed it?

Reply:
Yeah - good point, have added these.